bazel/linux/runner.bzl

```diff
@@ -132,13 +136,14 @@ def create_runner(ctx, archs, code, runfiles = None, extra = {}):
             ),
         )
 
-    prepares, runs, checks = get_prepare_run_check(ctx, ctx.attr.run)
+    prepares, runs, cleanups, checks = get_prepare_run_check(ctx, ctx.attr.run)
```

Comment:
This is also used in bazel/linux/test.bzl, do you need to update it there as well?

Reply:
Good catch. Thanks for that!